### DISCUSSION TITLE
posts/2023/03-10-markdown

### DISCUSSION THREAD

**Discussion by giscus** [L-TChen/L-TChen.github.io#10] [Comments]
# posts/2023/03-10-markdown

https://l-tchen.github.io/posts/2023/03-10-markdown

**Comment by L-TChen** [comment]
The inline math mode $\LaTeX$ and the displayed math mode
$$\int f(x)\mathrm{d}x$$
are also supported here. Some features are limited, e.g., the command `\,` is not supported.
